### PR TITLE
fastqc: import from homebrew/science

### PR DESCRIPTION
**Pull request by tmozgach**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fastqc.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Quality control tool for high throughput sequence data"
+  homepage "https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
+  url "https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.5.zip"
+  sha256 "dd7a5ad80ceed2588cf6d6ffe35e0f161c0d9977ed08355f5e4d9473282cbd66"
+
+  depends_on :java
+
+  def install
+    chmod 0755, "fastqc"
+    prefix.install Dir["*"]
```
Comment (fxcoudert) [inline Formula/fastqc.rb]: Install into `libexec`, not `prefix`.

File: Formula/fastqc.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    mkdir_p bin
```
Comment (fxcoudert) [inline Formula/fastqc.rb]: `mkdir_p` should not be necessary, IIRC `bin.install_symlink` will create the directory

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by tmozgach** [comment]
@fxcoudert  I took an account your changes.

**Comment by fxcoudert** [comment]
Does it need a specific or minimum Java version?

**Review by fxcoudert — APPROVED** [review]

**Comment by tmozgach** [comment]
Based on description (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/INSTALL.txt):
```
At the command prompt type 'java -version' and press enter.
If the version listed on the first
line is less than 1.6 then you might have problems running FastQC.
```

**Pull request merged**